src/Subscriber.ts

```diff
@@ -234,11 +234,13 @@ class SafeSubscriber<T> extends Subscriber<T> {
     if (!this.isStopped) {
       const { _parentSubscriber } = this;
       if (this._complete) {
+        const wrappedComplete = () => this._complete.call(this._context);
```

Comment:
@kwonoj that looks great! I like how it removes the branching in the `next` case